packages/paste-website/src/pages/blog/2022-10-31-paste-newsletter.mdx

```diff
@@ -0,0 +1,307 @@
+
+export const pageQuery = graphql`
+  {
+    mdx(frontmatter: {slug: {eq: "/blog/2022-10-31-paste-newsletter/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+        description
+        date(formatString: "MMMM D, YYYY")
+        machineDate: date
+        author
+        avatar
+        author_link
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ArticleHeader
+  title={props.data.mdx.frontmatter.title}
+  description={props.data.mdx.frontmatter.description}
+  date={props.data.mdx.frontmatter.date}
+  machineDate={props.data.mdx.frontmatter.machineDate}
+  author={props.data.mdx.frontmatter.author}
+  avatar={props.data.mdx.frontmatter.avatar}
+  authorLink={props.data.mdx.frontmatter.author_link}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} hideFeedback />
+
+<ArticleContent>
+
+## 📣 What's new in Paste
+
+### Radio Button Group
+
+![](../../assets/images/articles/2022-10-31-paste-newsletter/radio-button-group.png)
+
+Radio Buttons are here! This component is functionally similar to the Radio Group, but renders the controls visually as buttons instead of radios. They can be used either attached, like in the above screenshot, or unattached, for two stylistic varieties.
+
+<Paragraph>
+  <Anchor href="#/components/radio-button-group">Check out the Radio Button Group docs</Anchor>
+</Paragraph>
+
+### Button Group
+
+![](../../assets/images/articles/2022-10-31-paste-newsletter/button-group.png)
+
+The Button Group component is stylistically similar to the Radio Button Group, but doesn’t operate as a Radio Group. The Button Group is used for sets of related actions, rather than mutually exclusive options, that would otherwise be separate buttons.
+
+<Paragraph>
+  <Anchor href="/components/button-group">Check out the Button Group docs</Anchor>
+</Paragraph>
+
+### Side Modal
+
+![](../../assets/images/articles/2022-10-31-paste-newsletter/side-modal.png)
+
+Presenting Code Block! This component allows you to display code with syntax highlighting for 13 programming languages.
+It features tabs to show different blocks of code in the same area, a copy code button, and an optional external link button leading to the source code.
+
+<Paragraph>
+  <Anchor href="/components/code-block">Check out the Code Block docs</Anchor>
+</Paragraph>
+
+### File Picker
+
+![](../../assets/images/articles/2022-10-31-paste-newsletter/file-picker.png)
+
+Say goodbye to those boring, native `input type=”file”`s, and say hello to our styled File Picker! This component is actually the native HTML element under the hood, so you get all the same functionality, but we Pastified it so that it fits seamlessly into your form layouts.
+
+<Paragraph>
+  <Anchor href="/components/file-picker">Check out the File Picker docs</Anchor>
+</Paragraph>
+
+### Additional updates
+
+#### New
+
+- Inverse components in Figma (Anchor, Button, Combobox, Help Text, Input, and Select)!
```

Comment:
Question: is there a link for this?

Reply:
there wasn't one added to the newsletter 🤷